source/crud/tests/README.md

```diff
@@ -676,3 +676,15 @@ InsertOne {
 
 Execute `bulkWrite` on `client` with `model`. Assert that an error (referred to as `error`) is returned. Assert that
 `error` is a client error containing the message: "bulkWrite does not currently support automatic encryption".
+
+### 14. `explain` helpers allow users to specify `maxTimeMS`
+
+Create a MongoClient with command monitoring enabled (referred to as `client`).
+
+Create a collection, referred to as `collection`, with the namespace `explain-test.collection`.
+
+Run an explained find on `collection`. The find will have the query predicate `{ name: 'john doe' }`. Specify a
+maxTimeMS value of 2000ms for the `explain`.
```

Comment:
Maybe mention "repeat this test for every explain helper API". Drivers should make sure that this builds the correct command for their `cursor.explain({maxTimeMS})` and `find({}, {explain: {maxTimeMS}}` etc. if applicable. 

In that vein maybe you don't need to give an example command to be explained, but say check your outgoing command for all helpers?

Reply:
I'm happy to leave this vague and not specify the exact API drivers should use if you'd like.  However, prose tests do usually specify the exact commands and assertions, so my preference is to leave in the exact driver API and expect command.

Also, I don't believe exhaustiveness testing is something we should specify (and frankly, I think it's unnecessary - I didn't exhaustively test every API that supports explain in the Node driver).

Reply:
That sounds fine by me too, I called this out because the spec mentions both but the test does not, seems like an easy place to leave a test gap if we do not call it out here too.

Reply:
Good to resolve on my end